yarn-error.log

```diff
@@ -0,0 +1,93 @@
+Arguments: 
+  /Users/szymon.szajek/.nvm/versions/node/v12.13.0/bin/node /Users/szymon.szajek/.nvm/versions/node/v12.13.0/bin/yarn run dev
+
+PATH: 
+  /Users/szymon.szajek/.nvm/versions/node/v12.13.0/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Users/szymon.szajek/.nvm/versions/node/v12.13.0/bin
+
+Yarn version: 
+  1.19.1
+
+Node version: 
+  12.13.0
+
+Platform: 
+  darwin x64
+
+Trace: 
+  SyntaxError: /Users/szymon.szajek/Projects/Knot.x DevTool/fork/knotx-fragments-chrome-extension/package.json: Unexpected token < in JSON at position 271
+      at JSON.parse (<anonymous>)
+      at /Users/szymon.szajek/.nvm/versions/node/v12.13.0/lib/node_modules/yarn/lib/cli.js:1625:59
+      at Generator.next (<anonymous>)
+      at step (/Users/szymon.szajek/.nvm/versions/node/v12.13.0/lib/node_modules/yarn/lib/cli.js:304:30)
+      at /Users/szymon.szajek/.nvm/versions/node/v12.13.0/lib/node_modules/yarn/lib/cli.js:315:13
+
+npm manifest: 
+  {
+    "name": "knotx-chrome-devtool-extension",
+    "version": "1.0.0",
+    "description": "Extends the Developer Tools, adding a sidebar that displays the Knotx Fragments data associated with the selected DOM element.",
+    "private": true,
+    "scripts": {
+      "dev": "webpack",
+  <<<<<<< HEAD
```

Comment:
I think this fils shouldn't be commited. It should probably be added to git ignore